rust/sealevel/client/src/warp_route.rs

```diff
@@ -535,6 +634,21 @@ fn get_routers(
     Ok(token_data.remote_routers)
 }
 
+fn get_destination_gas(
+    client: &RpcClient,
+    token_program_id: &Pubkey,
+) -> Result<HashMap<u32, u64>, ClientError> {
+    let (token_pda, _token_bump) =
+        Pubkey::find_program_address(hyperlane_token_pda_seeds!(), token_program_id);
+
+    let account = client.get_account(&token_pda)?;
+    let token_data = HyperlaneTokenAccount::<()>::fetch(&mut &account.data[..])
+        .unwrap()
+        .into_inner();
+
+    Ok(token_data.destination_gas)
```

Comment:
mm good point, similar deal here where I'm refactoring a bunch of this in https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2502 (specifically have ripped out the router deployment logic so that helloworld programs and warp route programs can use it without code duplication). So if it's okay with you I'll make this change in #2502